opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/metric_reader_storage.py

```diff
@@ -114,6 +114,10 @@ def _get_or_init_view_instrument_match(
             return view_instrument_matches
 
     def consume_measurement(self, measurement: Measurement) -> None:
+
```

Comment:
How could it be None if it's not annotated as Optional?

Reply:
Removed these changes.